crates/guest-rust/src/examples/_1_interface_imports.rs

```diff
@@ -24,4 +24,9 @@ crate::generate!({
     "#,
 
     path: "wasi-cli@0.2.0.wasm",
+
+    // specify that this interface dependency should be generated as well.
+    with: {
+        "wasi:cli/environment@0.2.0": generate,
+    }
```

Comment:
Would it actually be possible to have a `dev-dependency` on the `wasi` crate to showcase `with` pointing to `wasi` in this situation?

Reply:
I believe dev-dependencies are not available for `#[cfg(docsrs)]` so it would have to be a regular dependency.

Reply:
Ah excellent point, in that case seems fine to leave this.